genie-web/src/main/java/com/netflix/genie/GenieWeb.java

```diff
@@ -41,8 +41,8 @@
 @EnableAspectJAutoProxy
 public class GenieWeb {
 
-    protected static final String SPRING_CONFIG_LOCATION = "spring.config.location";
-    protected static final String USER_HOME_GENIE = "${user.home}/.genie/";
+    static final String SPRING_CONFIG_LOCATION = "spring.config.location";
+    static final String USER_HOME_GENIE = "${user.home}/.genie/";
```

Comment:
Changes in this file are also not strictly related to this commit.
Splitting them into a different commit (and then not squashing on merge) would make the history of these lines more clear.